src/components/settings/SettingsLMSTab/tests/ExistingLMSCardDeck.test.jsx

```diff
@@ -67,6 +69,11 @@ describe('<ExistingLMSCardDeck />', () => {
     );
     expect(screen.getByText('Incomplete')).toBeInTheDocument();
     expect(screen.getByText('barfoo')).toBeInTheDocument();
+    act(() => {
```

Comment:
I don't think we require act around fireEvent calls, since their api already includes act

https://testing-library.com/docs/preact-testing-library/api/#fireevent